pymeasure/instruments/attocube/anc300.py

```diff
@@ -129,17 +135,40 @@ class Axis(Channel):
         "getc",
         """Measure the saved capacity value in nF of the axis.""")
 
+    continuous_move = Instrument.setting(
+        "%s",
+        """Set a continous movement in 'up' or 'down' direction. Mode must be
+        'stp'. The axis can be halted by the stop method.
+        """,
+        validator=strict_discrete_set,
+        values={"up": "stepu c", "down": "stepd c"},
+        map_values=True,
+        check_set_errors=True,
+    )
+
```

Comment:
I'd make it two methods (`move_continuously_up`, `move_continuously_down`) or add it to the `move` method, if the steps are +/- inf.

Reply:
I like the idea with the +/-inf. It seems unnecessary to have extra methods for this. I'll look into it.